src/quicknxs/interfaces/event_handlers/main_handler.py

```diff
@@ -718,6 +719,9 @@ def open_run_number(self, number=None):
             if not matches:  # Look for old-style nexus file name
                 search_string = configuration.instrument.legacy_search_template % run_number
                 matches = glob.glob(search_string + "_event.nxs")
+            if not matches:
+                self.report_message("Could not locate file %s" % search_string, pop_up=True)
```

Comment:
the search string is kind of ugly, what about just the run number?

```suggestion
                self.report_message("Could not locate run number %s" % run_number, pop_up=True)
```